spec/features/beis_users_can_create_a_programme_level_activity_spec.rb

```diff
@@ -257,24 +257,31 @@
       expect(created_activity.tags).to eq(non_oda_activity.tags)
     end
 
-    scenario "a new non-ODA programme can be linked to an existing ODA programme" do
-      linked_oda_activity = create(:programme_activity, :ispf_funded, is_oda: true, extending_organisation: partner_organisation)
-      non_oda_activity.linked_activity = linked_oda_activity
+    context "when the `activity_linking` feature flag is enabled" do
```

Comment:
Do you think it's worth adding tests here for the page being hidden when the flag is disabled for security here?

Reply:
I think this is covered multiple times by the existing specs because of how the helper works

```ruby
def fill_in_ispf_project_activity_form
    fill_in_identifier_step
    fill_in_linked_activity_step if ROLLOUT.active?(:activity_linking)
    fill_in_purpose_step
    fill_in_objectives_step if @activity.is_oda
    fill_in_sector_category_step
    fill_in_sector_step
    fill_in_call_details
    fill_in_call_applications
    fill_in_programme_status
    fill_in_dates
```

Reply:
Will do! Working in this now

Reply:
These are covered in the existing specs.

Reply:
Do you mean the activity policy ones?

Reply:
The linking specs are separate from the original "go through the form specs". So, we still have coverage for the journey without that specific page

I put the feature flag in the ActivityForm helper https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/a3954574841f927ca2798fdfb72debf6d8afb28d/spec/support/activity_form.rb#L132, so in [this spec](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/a3954574841f927ca2798fdfb72debf6d8afb28d/spec/features/beis_users_can_create_a_programme_level_activity_spec.rb#L233) the linking page is skipped.

Hope that makes sense

Reply:
Ah I see - that's fine then